src/components/DraggableBar.vue

```diff
@@ -1,7 +1,10 @@
 <template>
   <div class="playbar-clickable">
     <div :id="id" class="barbg" @mousedown="onMyMouseDown">
-      <div class="cur" :style="{ width: changingProgress ? cprogress * 100 + '%' : progress + '%' }">
+      <div
```

Comment:
i'm not sure why code formatter behavior different. Some rules control whether return line in html tag?

Reply:
I format it using prettier, it defined print width of 160 in .prettierrc which did extend the config from eslint.